Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
```

Comment:
The images already seem to be built with 1.12 in CI, except for 4.1

Reply:
ART's gitlab CI has its own config so you have to reach out to them to fix it.  Update release 4.1 to use 1.12 as well because we need to backport this.

Reply:
we should just pin the lib-go version to its appropriate version in the other branches IMHO, so bumping go should not be necessary (4.1 does not contain the offending tls1.3 constant)